docs/developer/extending/apps/architecture/app-permissions.mdx

```diff
@@ -0,0 +1,87 @@
+
+Permissions can also be modified after the app is installed. You can change them using the [appUpdate](api-reference/apps/mutations/app-update.mdx) mutation:
+
+```graphql
+mutation UpdateAppPermissions($id: ID!) {
+  appUpdate(id: $id, input: { permissions: [MANAGE_ORDERS] }) {
+    app {
+      permissions {
+        name
+        code
+      }
+    }
+  }
+}
+```
+
+## Using the app
+
+Apps are only visible for the users who have the `MANAGE_APPS` permission.
+
+:::info
+
+To install, deactivate or delete an app, the user must have the `MANAGE_APPS` permission, as well as all the app permissions.
+
+:::
+
+Let's consider an example with the Orders App, which sends orders to an external service, to differentiate between users who can use the app and those who can manage it. The app requires the `MANAGE_ORDERS` permission.
+
+A user who only has the `MANAGE_APPS` permission can open the installed app, but they won't be able to deactivate or delete it.
+
+In this case, only the user who has both the `MANAGE_APPS` and `MANAGE_ORDERS` permissions can install, deactivate or delete the app.
+
+## Access scopes
+
+When using an app, two types of access scope apply: user scope (which applies on the client-side) and app scope (which applies on the server-side).
+
+The user using the app is still restricted by their own permissions.
+
+Coming back to our Orders App example, let's imagine it features a table of orders. The table can be used to determine which orders have been synchronized with the external service.
+
+### User scope
+
+Our table is populated by the result of the [`orders` query](api-reference/orders/queries/orders.mdx). This query requires the `MANAGE_ORDERS` permission. The call is executed on the client side, where the user scope applies. It means that the user without the `MANAGE_ORDERS` permission will fail to execute the query.
+
+:::tip
+
+You can check if the user has the required permissions using the [`useAppBridge` hook](developer/extending/apps/developing-apps/app-sdk/app-bridge.mdx#useappbridge-hook). It passes both the user and the app context to the client-side.
+
+```jsx
+import { useAppBridge } from "@saleor/app-sdk";
+
+const Component = () => {
+  const { appBridgeState } = useAppBridge();
+
+  const hasPermission =
+    appBridgeState.user.permissions.includes("MANAGE_ORDERS");
+
+  return hasPermission ? (
+    <OrdersTable />
+  ) : (
+    <p>You don't have the required permissions</p>
+  );
+};
+```
+
+:::
+
+### App scope
+
+The app scope applies on the server-side of our app. It means that as long as the app has the `MANAGE_ORDERS` permission, it can execute the `orders` query.
+
+If you want to render the table for all the users, not just the ones with the `MANAGE_ORDERS` permission, you need to execute the orders query on the server-side and pass the result to the client-side.
```

Comment:
We can add some warning like "remember to implement authorization on the app-side, so only permitted staff members can perform allowed operations"

Reply:
Do you mean checking if the user has the right permissions to perform an operation? Because if so, I described it in the "tip" block above